tests/django/test_mocking.py

```diff
@@ -0,0 +1,230 @@
+from collections import defaultdict
+
+import pytest
+
+from schematools.contrib.django.db import create_tables
+from schematools.contrib.django.factories import (
+    schema_model_mockers_factory,
+    schema_models_factory,
+)
+from schematools.contrib.django.faker.create import create_data_for
+from schematools.contrib.django.faker.relate import relate_datasets
+from schematools.utils import to_snake_case
+
+from .conftest import DATABASE_URL
+
+
+@pytest.mark.django_db
+@pytest.mark.skipif(DATABASE_URL is None, reason="DATABASE_URL not set")
+def test_mocking_creates_data(gebieden_schema, gebieden_dataset):
+    """Prove that mocking create data."""
+    create_tables(gebieden_dataset)
+    create_data_for(gebieden_dataset, size=3)
+
+    table_schemas = {
+        to_snake_case(t.id): t for t in gebieden_schema.get_tables(include_through=True)
+    }
+    models = {}
+    for cls in schema_models_factory(gebieden_dataset, base_app_name="dso_api.dynamic_api"):
+        models[cls._meta.model_name] = cls
+
+    for model in models.values():
+        assert model.objects.count() == 3
+        # For composite keys, a faker automatically kicks in to create the proper `id`
+        if (table_schema := table_schemas[model._meta.model_name]).has_composite_key:
+            for obj in model.objects.all():
+                assert obj.id == ".".join(str(getattr(obj, fn)) for fn in table_schema.identifier)
+
+
+@pytest.mark.django_db
+@pytest.mark.skipif(DATABASE_URL is None, reason="DATABASE_URL not set")
+def test_mocking_take_min_max_into_account(
+    afvalwegingen_dataset, verblijfsobjecten_dataset, gebieden_dataset
+):
+    create_tables(gebieden_dataset)
+    create_tables(verblijfsobjecten_dataset)
+    create_tables(afvalwegingen_dataset)
+    create_data_for(afvalwegingen_dataset, size=30)
+
+    models = {}
+    for cls in schema_models_factory(afvalwegingen_dataset, base_app_name="dso_api.dynamic_api"):
+        models[cls._meta.model_name] = cls
+
+    # The containertypes.volume field has a `maximum` of 20 and `minimum` of 5
+    assert models["containertypes"].objects.filter(volume__gt=12, volume__lt=5).count() == 0
```

Comment:
20 or 12?

Reply:
Comment is not very useful anyway, I'll remove it.